.ci/scripts/prepare-benchmarks-env.sh

```diff
@@ -6,27 +6,47 @@
 # Usage:
 #   NODE_VERSION=...
 #   source .../prepare-benchmarks-env.sh
+#
+# Note: echo "--- ..." helps with presenting the output in Buildkite.
+#
 
 set -xeo pipefail
 
-if [[ -z "$NODE_VERSION" ]]; then
+if [[ -z "${NODE_VERSION}" ]]; then
   echo "prepare-benchmarks-env.sh: error: NODE_VERSION envvar is not set" >&2
   exit 1
 fi
 
+echo "--- Download nvm"
 # This particular configuration is required to be installed in the baremetal
 curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
-export NVM_DIR="$HOME/.nvm"
-set +x  # Disable xtrace because output using nvm.sh is huge.
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+export NVM_DIR="${HOME}/.nvm"
+
+echo "--- Install nvm"
+# set +x  # Disable xtrace because output using nvm.sh is huge.
+# Flush positional arguments
+shift $#
+if [ -s "${NVM_DIR}/nvm.sh" ] ; then
+  # As long as the Buildkite agent does some weird behaviours compare to the Jenkins agent
+  # let's avoid failures when running nvm.sh for the first time. For some reason
+  # nvm_resolve_local_alias default returns VERSION= while in Jenkins a similar
+  # runner returns VERSION=v14.21.3
+  # Already reported to the System owners of the Buildkite agent setup.
+  \. "${NVM_DIR}/nvm.sh"
```

Comment:
See https://github.com/elastic/apm-agent-nodejs/pull/3374/commits/597e18b37c5c91251935f6859dcec57123eee8ff


Pretty much I could not find the reason it failed when using the buildkite user in the bare-metal. I even raised an issue in the nvm-install, see https://github.com/elastic/apm-agent-nodejs/pull/3374#discussion_r1205231248

```suggestion
  \. "${NVM_DIR}/nvm.sh || true"
```

Otherwise, see https://github.com/elastic/apm-agent-nodejs/pull/3374#discussion_r1207277268 so we can move away from the nvm-install.

Just in case you get crazy with the nvm install errors...

Reply:
It seems to be related to the undefined NODE_VERSION env var and some crazy stuff behind the scene.